### DISCUSSION TITLE
Using R and then R-Instat for mapping spatial data

### DISCUSSION THREAD

**Discussion by rdstern** [IDEMSInternational/R-Instat#7524] [Ideas]
@N-thony asks "Hi, is it possible to get NDVI map with R-Instat?"

In looking for an answer I stumbled over the book called ["R as GIS for Economists](https://tmieno2.github.io/R-as-GIS-for-Economists/preface.html)".  

This goes further than I had ever thought.  I have found that climatologists often use GIS software and experts for quite simple mapping tasks.  For those tasks, I wondered (and still do), whether we could provide the facilities needed in R-Instat.  I wasn't thinking the more specialist software could be replaced.  

This book does.  In the preface it also discusses the alternatives, including using python, and arcgis and qgis.  So pretty interesting to consider the parallel for climatology - or even stick with the simple mapping tasks at least.

The key packages are almost all already in R-Instat.  So we already have the sf package and raster, and rgdal, though not yet rgeos or stars.

It also directed me to this [30-day map challenge](https://rud.is/books/30-day-map-challenge/).  This gives details of 30 different types of map and then draws them.  They are mainly for the US and particularly for Maine.  It could be interesting to see how many can be produced using R-Instat, changing US for Africa and Maine for Kenya?  Some are choropleths, so perhaps this could be a long-term challenge for @EstherNjeriLiberatta?  If so, then let's start with a more modest challenge, say aiming for a 5-day challenge?  The format is interesting in the book, so each challenge is a chapter, and perhaps we start with the choropleth examples?  And they don't have to be the same, so NDVI could be an example, perhaps.

So, within this area let's see what we can do about NDVI plots.  And let's start with the `Describe > General > Graphics ` as well as the dialogue for choropleths.

**Comment by N-thony** [comment]
@EstherNjeriLiberatta can you investigate on this? Thanks.

**Comment by N-thony** [comment]
@rdstern I think it it will be good for @MeSophie  to investigate this under @EstherNjeriLiberatta supervision?

**Comment by N-thony** [comment]
@MeSophie can you take this over?
